src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Machine {
```

Comment:
According to the best practices and encapsulation principle, consider using `public` or `protected` access modifier for the `Machine` class. This will ensure that the class can be extended by classes in other packages if necessary.